README.md

```diff
@@ -31,6 +31,8 @@ where `dependencies` is the list of dependencies divided by space, and `target`
 
 If no `target` argument passed, dependencies are written to `dependencies`.
 
+If no `packageFilePath` argument passed, dependencies are written to the default of `./package.json`, i.e /some/dir/from/root/package.json or ./../some/relative/path/package.json
```

Comment:
1. `./../some/relative/path/package.json` == `../some/relative/path/package.json`
1. Please, add markdown backtick inline quotes for file paths.

Reply:
addressed

Reply:
3. Maybe, the whole phrase starting with `i.e.` is redundant. Maybe we'd better say:

> If no `packageFilePath` argument passed, the script searches for a `package.json` file within the current directory.

?